packages/odf/components/s3-browser/bucket-details/BucketDetails.tsx

```diff
@@ -85,7 +100,7 @@ const getEncryptionDescription = (
   }
 };
 
-const BucketDetailsOverview: React.FC<{}> = ({}) => {
+const S3BucketDetailsOverview: React.FC<{}> = ({}) => {
```

Comment:
```suggestion
const S3BucketDetailsOverview: React.FC = () => {
```
Just curious why we need to use those curly braces?

Reply:
no particular reason, copy-paste from other files I guess (we are using that format other places too)... anything is fine (using and not using it)...